average/average.js

```diff
@@ -0,0 +1,65 @@
+function Array(value){
```

Comment:
This step isn't really necessary, but good on you for making a constructor function!